src/store.js

```diff
@@ -20,6 +19,7 @@ const store = configureStore({
     fraxData: fraxDataReducer,
     messages: messagesReducer,
   },
+  middleware: getDefaultMiddleware => getDefaultMiddleware({ serializableCheck: false }),
```

Comment:
What's this for?

Reply:
this is to disable the annoying error redux gives us in the dev console when we pass in svgs into the state

Reply:
this is needed for the images